modules/apps/object/object-test/src/testFunctional/tests/ObjectPersonalData.testcase

```diff
@@ -0,0 +1,109 @@
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		JSONUser.tearDownNonAdminUsers();
+
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-186452 - Verify that is possible to export personal data"
```

Comment:
`Verify a user's personal data can be exported`
I didn't correct this earlier, that's my mistake. 

Also, the ticket ID should be the story ticket and not the automation ticket